lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -144,6 +144,17 @@ describe('CategoryRenderer', () => {
 
     const audits = categoryDOM.querySelectorAll('.lh-audit');
     assert.equal(audits.length, category.auditRefs.length, 'renders correct number of audits');
+
+    assert.equal(categoryDOM.querySelector('.lh-gauge__wrapper--plugin'), null);
+  });
+
+  it('plugin category has plugin badge', () => {
+    const category = JSON.parse(
+      JSON.stringify(sampleResults.reportCategories.find(c => c.id === 'seo')));
+    // Any category not of the core
+    category.id = 'lighthouse-plugin-someplugin';
```

Comment:
also needs the key and `id` to match, so maybe just append a fake category? Just need a `id`/`title`/`score`/empty `auditRefs`

Reply:
There's no key here.

Reply:
> There's no key here.

er, yeah, whoops :)